datafusion/proto/src/physical_plan/mod.rs

```diff
@@ -477,30 +477,10 @@ impl AsExecutionPlan for protobuf::PhysicalPlanNode {
                             ExprType::AggregateExpr(agg_node) => {
                                 let input_phy_expr: Vec<Arc<dyn PhysicalExpr>> = agg_node.expr.iter()
                                     .map(|e| parse_physical_expr(e, registry, &physical_schema, extension_codec)).collect::<Result<Vec<_>>>()?;
-                                let ordering_req: Vec<PhysicalSortExpr> = agg_node.ordering_req.iter()
+                                let _ordering_req: Vec<PhysicalSortExpr> = agg_node.ordering_req.iter()
```

Comment:
this is an interesting change -- does it mean ordering is not carried into the udf? 

Or maybe it is redundant and now is entirely determined by the udf

Reply:
Ordering are not supported for udaf yet. I had leave a TODO comment below

Reply:
Thanks

Filed https://github.com/apache/datafusion/issues/11804 / https://github.com/apache/datafusion/pull/11805 to track